roles/postgresql/tasks/backup_db.yml

```diff
@@ -0,0 +1,13 @@
+---
+# back up the database to /tmp/ on the app's VM (not the database VM)
+- name: backup postgres database
+  postgresql_db:
+    name: "{{ application_db_name }}"
+    login_host: "{{ postgres_host }}"
+    port: "{{ postgres_port  }}"
+    login_user: "{{ vault_postgres_admin_user }}"
+    login_password: "{{ vault_postgres_admin_password }}"
+    encoding: "UTF-8"
+    owner: "{{ application_dbuser_name }}"
+    state: "dump"
+    target: "{{ db_backup }}"
```

Comment:
I think the current db backup puts content in tmp, but that's not ideal. Any problem with putting it under the deploy user somewhere? (could be a later improvement)